### PR TITLE
Feature/horizontal timeline v2

### PR DESCRIPTION
**Pull request by zJuarez**
### What does this PR do?

- new horizontal timeline implementation (responisve -> 3 elements on large, 2 on medium)
- on mobile devices still previous vertical timeline
- added common hook useWindowSize

** waiting on colors and general design approval**

#### Related issues

#118
image
image

image
image

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] No code style issues (eslint and flow)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,6 @@ yarn-error.log*
 
 .bash_history
 /docker-compose.override.yml
+
+# Other Files
+package-lock.json
```
Comment (AuroTB) [inline .gitignore:31]: thanks!

File: src/data/timeline.json [hunk 2]
```diff
@@ -1,145 +1,194 @@
 {
+  "years" : [2015, 2016, 2017,2018,2019,2020],
```
Comment (AuroTB) [inline src/data/timeline.json:2]: nice!

File: src/hooks/useWindowSize.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default function useWindowSize() {
+  const isSSR = typeof window !== 'undefined'
+  const [windowSize, setWindowSize] = React.useState({
+    width: isSSR ? 1200 : window.innerWidth,
```
Comment (AuroTB) [inline src/hooks/useWindowSize.js:6]: this is very nice

File: src/components/About/AboutTimeline/AboutTimeline.js [hunk 4]
```diff
@@ -1,23 +1,45 @@
...
+  const { events, years } = props
+  const { width } = useWindowSize()
+
+  if (width > SMALL_WIDTH) {
+    return (
+      <div className="about-timeline-container" test-id="1">
+        <Container fluid>
+          <Row className="justify-content-md-center" id="timeline-title">
+            <h1>
+              Our Story
```
Comment (AuroTB) [inline src/components/About/AboutTimeline/AboutTimeline.js:35]: you can add responsivity for thsi title too, looks very smile in [some very large views](https://github.com/RoBorregos/roborregos-web/wiki/Testing#responsivity)

image

**Review by AuroTB — CHANGES_REQUESTED** [review]
Responsivity in this component is extremely well done!
I pointed some UI stuff (colors to #FF7549, in small and large view), but other than that the code looks clean and nice.

image
(Change to orange)

File: src/components/About/About.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ const About = () => {
   return (
     <>
       <AboutHeader />
-      <AboutTimeline events={timelineData.events} />
+      <AboutTimeline events={timelineData.events} years={timelineData.years} />
```
[the remark this answers is not in the document]
Comment (zJuarez) [reply src/components/About/About.js:13]: import?

**Review by AuroTB — APPROVED** [review]
LGTM Great job!

**Pull request merged**